about.html

```diff
@@ -0,0 +1,123 @@
+        <div class="headerLogout"><a href="">Logout</a></div>
+      </div>
+
+      <nav>
+        <div class="navLogo"><img src="images/mclarenlogo..png"></div>
+        <div class="nav"><a href="about.html">About</a></div>
+        <div class="nav"><a>Program</a></div>
+        <div class="nav"><a>Join</a></div>
+        <div class="nav"><a>Sponsor</a></div>
+        <div class="nav"><a>News</a></div>
+        <div class="navCampaign"><a>Campaign</a></div>
+      </nav>
+    </header>
+
+    <main>
+     
+      <section id="landing">
+        
+        <div class="landing-container">
+
+          <div class="menu-mobile" ><i class="fa-solid fa-bars" id="trigger"></i></div>
+
+          <div id="menumobile">
+            <div class="menuClose"><i class="fa-solid fa-square-xmark" id="menuClose"></i></div>
+            <ul>
+              <li><a href="#works" id="menuOption1">Portfolio</a></li>
+              <li><a href="#about" id="menuOption2">About</a></li>
+              <li><a href="#contact" id="menuOption3">Contact</a></li>
+            </ul>
+          </div>
+
+          <div class="landing-subTitle"><p>McLAREN ESPORTS</p></div>
+
+          <div class="landing-title"><h1>Welcome to McLaren virtual racing programme. </h1></div>
+
+          <div class="landing-text"><p>McLaren Plus is our free-to-join fan loyalty programme,
+             bringing McLaren fans closer to the team with the most inclusive, rewarding and open-to-all fan programmes in F1 & esports. 
+            Sign up now, or current members can amend their details in the form below if necessary. 
+            Because, unlike any other sport,
+            the transference of skills between virtual and real racing are both direct and profound.</p></div>
+      
+          <div class="landing-location"><p>Don't miss out! We want you to be the first to know about competitions, news, offers and much more.
+            <br> shadowproject@mclaren.com
+          </p></div>
+
+        </div>
+
+      </section>
+
+      <section id="section2">
+        <div class="section2-container">
+            <div><h2>F1 2022 E-Sports Championship</h2></div>
+            <div><p>Attempt to qualify for the F1® Esports Series by registering below
+                 and you could become an official F1 Esports driver for an official
+                  F1 team, competing in the F1 Esports Pro Championship.</p>
+            </div>
+            <div><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/3/33/F1.svg/300px-F1.svg.png"></div>     
+        </div>
+      </section>
+
+      <section id="section3">
+        <div class="section2-container">
+            <div><h2>F1 2022 E-Sports McLaren Drivers</h2></div>
+            <div><p>Racing for McLaren E-Sports since 2019, he is our young talents.</p>
+            </div>
+            <div class="driver1"><h2>Mike Turner</h2></div>
+            <div class="driver2"><h2>Matt Helders</h2></div>
```

Comment:
- [ ] Please in your desktop view, kindly arrange these images horizontally to match the design guideline.
 ### Your design
![2](https://user-images.githubusercontent.com/94551484/192071744-1a745226-45c4-4c9a-a91d-c515da80fdfc.png)
### Design guidelines
![f2](https://user-images.githubusercontent.com/94551484/192071769-939af015-ad3c-4851-ac22-2a60b69fc202.png)